src/Microsoft.ML.ImageAnalytics/ImageResizer.cs

```diff
@@ -381,7 +382,15 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                             destHeight = info.ImageHeight;
                         }
 
-                        dst = new Bitmap(info.ImageWidth, info.ImageHeight, src.PixelFormat);
+                        if (src.PixelFormat == PixelFormat.Format1bppIndexed ||
```

Comment:
This check can be simplified using the same check from the corefx code:

https://source.dot.net/#System.Drawing.Common/System/Drawing/Graphics.Windows.cs,138

```C#
 if ((image.PixelFormat & PixelFormat.Indexed) != 0)
```

Reply:
Thanks, @eerhardt ! you are the best!